bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -81,7 +83,12 @@ public class ImageImpl extends com.adobe.cq.wcm.core.components.internal.models.
     /**
      * The width variable to use when building {@link #srcUriTemplate}.
      */
-    private static final String SRC_URI_TEMPLATE_WIDTH_VAR = "{.width}";
+    static final String SRC_URI_TEMPLATE_WIDTH_VAR = "{.width}";
+
+    /**
+     * The width variable to use when building {@link #srcUriTemplate} for CDN route.
+     */
+    static final String SRC_URI_TEMPLATE_WIDTH_VAR_ASSET_DELIVERY = "{width=width}";
```

Comment:
I would have prefer we add `width={width}` instead

Reply:
done